<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rao's Xtension Box</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="PrimaryGroupBox">
         <property name="title">
          <string>Primary key </string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Column:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="PrimaryColumnCombo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Order:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="PrimaryOrderCombo">
                <item>
                 <property name="text">
                  <string>Ascending </string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Descending </string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="OkButton">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Morebutton">
           <property name="text">
            <string>&amp;More</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="SecondaryGroupBox">
         <property name="title">
          <string>Secondary key</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Column:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="SecondayColumnCombo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Order:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="SecondaryOrderCombo">
                <item>
                 <property name="text">
                  <string>Ascending </string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Descending </string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="TertiaryGroupBox">
         <property name="title">
          <string>Tertiary key</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Column:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="TertiaryColumnCombo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Order:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="TertiaryOrderCombo">
                <item>
                 <property name="text">
                  <string>Ascending </string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Descending </string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>OkButton</zorder>
  <zorder>CancelButton</zorder>
  <zorder>Morebutton</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>PrimaryGroupBox</zorder>
  <zorder>SecondaryGroupBox</zorder>
  <zorder>TertiaryGroupBox</zorder>
  <zorder>verticalSpacer_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>PrimaryColumnCombo</tabstop>
  <tabstop>PrimaryOrderCombo</tabstop>
  <tabstop>SecondayColumnCombo</tabstop>
  <tabstop>SecondaryOrderCombo</tabstop>
  <tabstop>TertiaryColumnCombo</tabstop>
  <tabstop>TertiaryOrderCombo</tabstop>
  <tabstop>OkButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>Morebutton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Morebutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>SecondaryGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Morebutton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TertiaryGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
